deploy/k8s/standalone-agent.yml

```diff
@@ -7,45 +7,422 @@ metadata:
     k8s-app: elastic-agent
```

Comment:
I couldn't find usage for this file? is it for local testing? if so, how can I test it?

Reply:
It's exactly for that. You  can test it by running kubectl applying to your local cluster.